app/views/vendors/edit.html.erb

```diff
@@ -0,0 +1 @@
+<%= render 'vendorinput' %>
```

Comment:
You've been using underscores between muti-word variable/file names until now. It's helpful if you stick to one convention throughout a project.